Userland/Libraries/LibSQL/CMakeLists.txt

```diff
@@ -7,24 +7,25 @@ set(SOURCES
     AST/Parser.cpp
     AST/Select.cpp
     AST/Statement.cpp
-        AST/SyntaxHighlighter.cpp
-        AST/Token.cpp
-        BTree.cpp
-        BTreeIterator.cpp
-        Database.cpp
-        HashIndex.cpp
-        Heap.cpp
-        Index.cpp
-        Key.cpp
-        Meta.cpp
-        ResultSet.cpp
-        Row.cpp
-        Serializer.cpp
-        SQLClient.cpp
-        TreeNode.cpp
-        Tuple.cpp
-        Value.cpp
-        )
+    AST/SyntaxHighlighter.cpp
```

Comment:
Ah looks like you fixed the spacing here, that should be squashed into the previous commit

Reply:
CLion annoys me sometimes. I did the rebase dance already and thought I'd fixed it. It looks fine in the editor, I commit, and it's all screwed up again. I should invest some time teaching it proper indentation of CMake files.